pkg/querier/queryrange/queryrangebase/roundtrip.go

```diff
@@ -159,8 +160,21 @@ func (q roundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
 	return q.codec.EncodeResponse(r.Context(), response)
 }
 
+type roundTripperHandler struct {
+	next  http.RoundTripper
+	codec Codec
+}
+
+// NewRoundTripperHandler TODO.
```

Comment:
What is the todo here?

Reply:
I had to add some docs to that function. Done.